### PR TITLE
Dataset description additions

### PR DESCRIPTION
**Pull request by msleckman**
Dataset description additions and metadata information

### CODE REVIEW

File: in_text/text_02_observations.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 title: >-
-  Data release to support multiple modeling efforts in the Delaware River Basin: Observations XX
+  Data release to support multiple modeling efforts in the Delaware River Basin: Observations 
 
 abstract: >-
   <p>XX [MODIFY, leaving old text in here for an example] Observations related to water and thermal budgets in the Delaware River Basin. Data from reservoirs in the basin include
```
Comment (msleckman) [inline in_text/text_02_observations.yml:5]: @limnoliver I have not edited teh abstract for the observations. I'll let you that on.

File: in_text/text_03_driver.yml [hunk 2]
```diff
@@ -367,30 +364,30 @@ entities:
       data-min: NA
       data-max: NA
       data-units: NA
-    -
-      attr-label: model_idx
```
Comment (msleckman) [inline in_text/text_03_driver.yml:371]: this param is not a variable in the uncal_sntemp nc file. So I removed. Is that expected?

File: in_text/text_03_driver.yml [hunk 3]
```diff
@@ -405,8 +402,8 @@ entities:
       attr-def: >-
         Streamflow leaving a segment
       attr-defs: This data release
-      data-min: 1.50589e-05
-      data-max: 838.4617
+      data-min: 0
+      data-max: 3086.54
       data-units: cubic meters per second
     -
       attr-label: seg_rain
```
Comment (msleckman) [inline in_text/text_03_driver.yml:409]: Many of the data ranges for these variables were very different from the min and max documented last year. 

ex (units (m) ): 
```
sntemp.seg_rain.max()
Out[16]: 
<xarray.DataArray 'seg_rain' ()>
array(1587.5)` 
```
vs last year, max was 0.16 
```
attr-label: seg_rain
      attr-def: >-
        Area-weighted average rainfall for each segment from HRUs contributing flow to the segment
      attr-defs: https://doi.org/10.1002/joc.3413
      data-min: 0.0
      data-max: 0.160782
```

Comment (msleckman) [reply in_text/text_03_driver.yml:409]: @jzwart have you took a look at the mins and max for uncal_sntemp vs last year's ?

Comment (jzwart) [reply in_text/text_03_driver.yml:409]: No I haven't had the chance. How often are there large numbers for rain, etc...? once, twice, or many times?

File: in_text/text_03_driver.yml [hunk 4]
```diff
@@ -352,7 +349,7 @@ entities:
       contributing hydrologic reach units and/or upstream segments can have streamflow but
       may not have streamflow at the current timestep (-98.9)
```
Comment (msleckman) [inline in_text/text_03_driver.yml:351]: This `data-description` is last year's text. 

@jzwart Can you read through this text and update to how to the uncal_sntemp_input_out.nc.zip is built?

File: in_text/text_03_driver.yml [hunk 5]
```diff
@@ -113,237 +106,241 @@ entities:
...
-      sntemp_inputs_outputs_{subset}.zip
+      uncal_sntemp_inputs_outputs.nc.zip
     data-description: >-
       PRMS-SNTemp inputs and outputs, including weather driver data, intermediate varibles, and
       model predictions. There are two files; files either contain inputs/outputs from all 456 stream reaches in the
       Delaware River Basin modeling network, or for the 56 segments above Lordville, New York.
 
-      
+
       Code used to generate these inputs and outputs can be found in the repository "run-prms-sntemp" in Zwart and others 2021 (https://doi.org/10.5281/zenodo.5164910).
```
Comment (jzwart) [inline in_text/text_03_driver.yml:341]: This DOI will change to this year's code repo, but it hasn't been created yet. This old code repo should be ok and still has run-prms-sntemp code in it.

File: in_text/text_03_driver.yml (hunk 4 above)
Comment (jzwart) [reply in_text/text_03_driver.yml:351]: Description looks good, other than changing the DOI for code repo

File: in_text/text_03_driver.yml (hunk 2 above)
Comment (jzwart) [reply in_text/text_03_driver.yml:371]: Yes that's ok to leave out. We don't use `model_idx` for anything and it is cutout-specific so not very useful to other modeling applications outside of this specific cutout of the DRB. Relying on `seg_id_nat` is better

File: in_text/text_03_driver.yml [hunk 6]
```diff
@@ -367,30 +364,30 @@ entities:
...
       data-units: NA
     -
       attr-label: seg_ccov
       attr-def: >-
         Area-weighted average cloud cover fraction for each segment from HRUs contributing flow to the segment.
       attr-defs: https://doi.org/10.1002/joc.3413
-      data-min: 0.2565
-      data-max: 0.8
+      data-min: 0.2542
+      data-max: 7667
```
Comment (jzwart) [inline in_text/text_03_driver.yml:390]: this is quite high and should be between 0 and 1 if a decimal fraction. Do you know when and for what segment this is occurring ?

File: in_text/text_03_driver.yml [hunk 7]
```diff
@@ -405,8 +402,8 @@ entities:
       attr-def: >-
         Streamflow leaving a segment
       attr-defs: This data release
-      data-min: 1.50589e-05
-      data-max: 838.4617
+      data-min: 0
+      data-max: 3086.54
```
Comment (jzwart) [inline in_text/text_03_driver.yml:406]: similar to the high `ccov` number, it'd be good to see when and where this is occuring

File: in_text/text_03_driver.yml [hunk 8]
```diff
@@ -113,237 +106,242 @@ entities:
...
+  #     data-units: NA
+  #
   -
     data-name: >-
-      sntemp_inputs_outputs_{subset}.zip
+      uncal_sntemp_inputs_outputs.nc.zip
     data-description: >-
       PRMS-SNTemp inputs and outputs, including weather driver data, intermediate varibles, and
       model predictions. There are two files; files either contain inputs/outputs from all 456 stream reaches in the
       Delaware River Basin modeling network, or for the 56 segments above Lordville, New York.
```
Comment (msleckman) [inline in_text/text_03_driver.yml:339]: @limnoliver  does this 56 segments call out for Lordville apply here?

Comment (msleckman) [reply in_text/text_03_driver.yml:339]: Responded on teams, will edit this text in there.

**Comment by msleckman** [comment]
Think this is ready for review/merge. A couple XX left for pending additions (DOI link)

**Pull request merged**